src/app/zones/sources.ts

```diff
@@ -76,6 +78,14 @@ export const sources = (api: KumaApi) => {
       return api.getZoneIngressData({ zoneIngressName: name, dataPath })
     },
 
+    '/zone-ingress-overviews/:name': async (params: DetailParams, source: Closeable) => {
```

Comment:
How comes the listing URIs for these are just `/zone-ingresses` and the details ones are `/zone-ingress-overviews`.

Is there a reason why the following would work out?

```
`/zone-ingresses`
`/zone-ingress/:name`
```

i.e. use the following everywhere

```
/things
/thing/:name
```

Maybe I'm not following what is being returned in these sources? lmk

Reply:
Ah I should read the description more carefully 🤦

Reply:
I updated the list entries to also include `-overviews` to make this unambiguous.